internal/monitors/manager.go

```diff
@@ -105,7 +105,7 @@ func (mm *MonitorManager) Configure(confs []config.MonitorConfig, collectdConf *
 		if err != nil {
 			log.WithFields(log.Fields{
 				"monitorType": conf.Type,
-				"error":       err,
+				"error":       err.Error(),
```

Comment:
I changed this to `.Error()` because it was including tracebacks from `Wrapf` calls.

Reply:
Yeah I don't even use that Wrapf thing anymore, the errors are too verbose.

Reply:
Too verbose due to the stack trace or due to it printing the chain of error strings?

Reply:
The stack trace is too long and unnecessary for end users.